### DISCUSSION TITLE
Translation (or more) for large texts such as entire books

### DISCUSSION THREAD

**Discussion by hydropix** [oobabooga/text-generation-webui#1561] [Ideas]
LLMs are excellent for translation and summarization tasks. I would like to be able to automatically process a sequence of small paragraphs from a larger text like a book. This plugin should be able to extract groups of sentences (not cut the sentences) and repeat the operation until you read the whole content and save the translation or the summary (or something else) in a separate document. There should be a prompt sentence followed by the document part, like: "Translate this text into English": (...)

**Comment by BarfingLemurs** [comment]
This would be less advisable than using an expert translation model, as the models may hallucinate and aren't going to be 100% accurate.

**Reply by sarutobiumon** [reply]
But the whole point of oobabooga is to allow anyone to use any LLM model, including specialized ones, using a user-friendly UI. In the future, the UI will probably be redesigned around model use cases rather than model names.

**Reply by Agiltech-com-au** [reply]
would be great if we could at least just batch job an input, with output exported to txt.
